Dockerfile

```diff
@@ -0,0 +1,55 @@
+FROM bioconductor/bioconductor_docker:3.17 as bioconductor
```

Comment:
create a directory named "extras" then put everything related to tests and benchmark into it.